chore: Miscellaneous improvements to ERTP API pages (#916)
Agoric/documentation: main/reference/ertp-api/amount-math.md

Creates an **Amount** from a given **Brand** and **AmountValue**.

```js
const bid = AmountMath.make(quatloosBrand, 300n);
```

## AmountMath.coerce(brand, allegedAmount)
- **brand**: **[Brand](./brand.md)**
- **allegedAmount**: **[Amount](./ertp-data-types.md#amount)**
- Returns: **Amount**

> Verifies that an **Amount** is for the specified *brand* and returns a new equivalent **Amount**.

Reviewer: ```suggestion
Verifies that an **Amount** is for the specified *brand* and returns an equivalent **Amount**.
```

The PR overcorrects the original. The original mistakenly implied it is the same amount object. The PR mistakenly states it is guaranteed to be a fresh amount object, even if the original would have been fine. I think the API spec should only guarantee equivalence, without stating whether or not it is a new JS allocation.

- **brand**: **[Brand](./brand.md)** - Optional, defaults to **undefined**.
- Returns: **Amount**

Returns a new **Amount** that is the combination of *leftAmount* and *rightAmount*.
Both **Amount** arguments must have the same **Brand**.

If the optional *brand* argument doesn't match the **Amount**s' **Brand**, an error is thrown.

For fungible **Amounts** this means adding their **[AmountValues](./ertp-data-types.md#amountvalue)**. For non-fungible
**Amounts**, it usually means including all of the elements from *leftAmount*
and *rightAmount*.

If one of *leftAmount* or *rightAmount* is empty, this method returns an **Amount** equivalent to the other.

> If both are empty, this method returns a new empty **Amount**.

Reviewer: ```suggestion
If both are empty, this method returns an empty **Amount**.
```

Likewise. The API spec should not say whether or not the empty amount is newly allocated or not.
